pootle/static/css/editor.css

```diff
@@ -1728,3 +1734,16 @@ td.translate-delete-terminology input
 {
     float: right;
 }
+
+
+.js-auto-matched-translation textarea
+{
+    color: #aaa;
+}
+
+.js-auto-match-msg
```

Comment:
its used by js also - and other places do this already

it saves adding more classes